assets/_scss/default/body-main-styles.scss

```diff
@@ -389,6 +390,9 @@ $border-color: #a1a1a1;
   font-size: var(--text-size);
   margin-left: 25px;
 }
+#offering-services div {
```

Comment:
It would be better to refactor and create a separate `scss` with all the changes for the offerings page. There's already a section for them:
https://github.com/curacultura/curacultura-website/blob/c6e36dc352ba6c3909e68c88059eb1d5fb4c90a6/assets/_scss/default/body-main-styles.scss#L93